sqlmesh/core/config/connection.py

```diff
@@ -84,6 +85,7 @@ def create_engine_adapter(self) -> EngineAdapter:
             cursor_kwargs=self._cursor_kwargs,
             default_catalog=self.get_catalog(),
             cursor_init=self._cursor_init,
+            register_comments=register_comments or self.register_comments,
```

Comment:
If `None` means `False` anyway why not just use `False` as a default argument value.